.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## Description
+
+*Describe problem and desired outcome.*
+
+## Solution Ideas
```

Comment:
Before this we need something like: Minimal reproduction of the problem with instructions